berserk/clients/account.py

```diff
@@ -60,3 +61,17 @@ def upgrade_to_bot(self):
         """
         path = "/api/bot/account/upgrade"
         self._r.post(path)
+
+    def get_timeline(self, since: int = 1356998400070, nb: int = 15) -> TimelineEvents:
+        """Get your timeline events.
+
+        Requires OAuth2 authorization.
+
+        :param int since: timestamp to show events since, default 1356998400070
+        :param int nb: max number of events to fetch, default 15
```

Comment:
```suggestion
        :param since: only show events after this timestamp, default 1356998400070 (the earliest allowed by Lichess)
        :param nb: max number of events to fetch, default 15
```

We generally leave out the types here since they are already given via the type hints.